src/main/java/sirius/biz/importer/format/FieldDefinition.java

```diff
@@ -264,6 +265,25 @@ public FieldDefinition markRequired() {
         return withCheck(new RequiredCheck());
     }
 
+    /**
+     * Marks the current field definition to be included in error messages with name and value for row identification purposes.
+     *
+     * @return the field itself for fluent method calls
+     */
+    public FieldDefinition appendContextInErrorMessage() {
+        this.appendContextInErrorMessage = true;
+        return this;
+    }
+
+    /**
+     * Returns the info if this field's name and value should be included in error messages.
+     *
+     * @return true if this field's name and value should be included in error messages, false otherwise
+     */
+    public boolean isAppendContextInErrorMessage() {
```

Comment:
Now this method's name look kinda strange. shouldAppend... fits better its purpose, but let's see what the others have to say.

Reply:
has been renamed to shouldAppendContextInErrorMessage